### PR TITLE
Wp e2e playwright

### PR DESCRIPTION
**Pull request by imranhasanraaz**

### CODE REVIEW

File: tests/wp-e2e-playwright/specs/media_size.spec.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { test, expect } from "@wordpress/e2e-test-utils-playwright";
+import Activity from "../page_model/activity.js";
```
Comment (pooja-muchandikar) [inline tests/wp-e2e-playwright/specs/media_size.spec.js:2]: @imranhasanraaz Move this file into the Other Settings folder.

File: tests/wp-e2e-playwright/testdata/images.jpg
Comment (pooja-muchandikar) [inline tests/wp-e2e-playwright/testdata/images.jpg:1]: @imranhasanraaz  you are fetching this media from the testdata repo then still why you have uploaded these images? Remove it if not required.

File: tests/wp-e2e-playwright/utils/solveCaptcha.js
Comment (pooja-muchandikar) [inline tests/wp-e2e-playwright/utils/solveCaptcha.js:1]: This file is not required remove it

**Review by pooja-muchandikar — COMMENTED** [review]
@imranhasanraaz 

Suggested a few changes. Remove all unwanted files.

**Review by pooja-muchandikar — COMMENTED** [review]
@imranhasanraaz 

CI is failing. Take a look at it.

File: tests/wp-e2e-playwright/utils/solveCaptcha.js
Comment (imranhasanraaz) [reply tests/wp-e2e-playwright/utils/solveCaptcha.js:1]: Solved the above feedback @pooja-muchandikar

**Comment by pooja-muchandikar** [comment]
@imranhasanraaz 

Remove the [playwright-report](https://github.com/rtCamp/rtMedia/tree/wp-e2e-playwright/tests/wp-e2e-playwright/playwright-report) folder. Rest looks fine.

**Comment by imranhasanraaz** [comment]
@pooja-muchandikar removed playwright-report folder

**Review by pooja-muchandikar — APPROVED** [review]

**Pull request merged**